src/WebJobs.Script/Host/FunctionMetadataProvider.cs

```diff
@@ -20,16 +18,15 @@ namespace Microsoft.Azure.WebJobs.Script
 {
     public class FunctionMetadataProvider : IFunctionMetadataProvider
     {
-        private readonly IEnumerable<WorkerConfig> _workerConfigs;
         private readonly IOptionsMonitor<ScriptApplicationHostOptions> _applicationHostOptions;
         private readonly IMetricsLogger _metricsLogger;
         private readonly Dictionary<string, ICollection<string>> _functionErrors = new Dictionary<string, ICollection<string>>();
         private readonly ILogger _logger;
         private ImmutableArray<FunctionMetadata> _functions;
+        private Dictionary<string, JObject> _functionConfigs = new Dictionary<string, JObject>();
 
-        public FunctionMetadataProvider(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, IOptions<LanguageWorkerOptions> languageWorkerOptions, ILogger<FunctionMetadataProvider> logger, IMetricsLogger metricsLogger)
+        public FunctionMetadataProvider(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILogger<FunctionMetadataProvider> logger, IMetricsLogger metricsLogger)
```

Comment:
@soninaren  - Please take a look at changes in FunctionMetadataProvider  and FunctionMetadataManager